test/unittest/layers/meson.build

```diff
@@ -58,8 +58,3 @@ exe = executable(
   install: get_option('enable-test'),
   install_dir: application_install_dir
 )
-
-# @todo delete v2 once migration is done
-test('unittest_layers_v2', exe,
```

Comment:
Maybe s/unittest_layers_v2/unittest_layers/ ?

I think deleting this might remove test runs from gbs, pdebuild.

Reply:
The tests built in this are tested using this statement I think.
So, I think rename should do.

Reply:
Verified locally, deleting this disables this test runs.
Just rename is required.